### ISSUE TITLE
 Show messages when rejecting the --env.snapshot option

### ISSUE THREAD

**Issue by vakrilov** [NativeScript/nativescript-dev-webpack#441]
For example if running on windows where snapshot generation is not available.

**Comment by NickIliev** [comment]
Implemented with the current next version
```
Running Webpack for Android...
Stripping the snapshot flag. Bear in mind that snapshot is only available in release builds and is NOT available on Windows systems.
```

**Comment by sis0k0** [comment]
Resolved with #448
